paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/UpdatedDataFieldsProcessFunctionBase.java

```diff
@@ -97,7 +97,7 @@ protected void applySchemaChange(
                             e);
                 }
             }
-        } else if (schemaChange instanceof SchemaChange.UpdateColumnType) {
+            //        } else if (schemaChange instanceof SchemaChange.UpdateColumnType) {
```

Comment:
Why modify here?

Reply:
Must be a typo. Correct with later commit.